### ISSUE TITLE
Poster size

### ISSUE THREAD

**Issue by fasteater** [google/model-viewer#1147]
Can I get a larger poster size? seems like it is related to the viewer size.

**Comment by elalish** [comment]
@fasteater I'm guessing you're talking about the posters generated by this [page](https://modelviewer.dev/examples/tester.html)? They are based on the size of the viewport. It just uses our public `toBlob` method, so you should be able to use it with your own Glitch to make any poster you like. 

Out of curiosity, why do you want a larger poster than fits on your monitor? Seems like that could take a relatively long time to download.

**Comment by fasteater** [comment]
@elalish thanks for the reply. I figured out the poster size depends on the size of the model-viewer, I managed to do a quick hack in js to enlarge the model-viewer before calling `toBlob()` then quickly shrink the model-viewer back original size.

> Out of curiosity, why do you want a larger poster than fits on your monitor? Seems like that could take a relatively long time to download.

I want to add model-viewer to my little pet project https://www.isometriclove.com/, so user can choose desired render angle and download a high rez poster for their design needs.
